scipy/interpolate/tests/test_interpolate.py

```diff
@@ -913,7 +913,7 @@ def test_axis(self):
         c_s = c.shape
         xp = np.random.random((1, 2))
         for axis in (0, 1, 2, 3):
-            k, m = c.shape[axis], c.shape[axis+1]
+            _, m = c.shape[axis], c.shape[axis+1]
```

Comment:
Ditto, if c.shape[axis] is not used, then drop it.

Reply:
Done.